### DISCUSSION TITLE
Can nginx-lua-plugin set the sample rate?[Feature]

### DISCUSSION THREAD

**Discussion by HackerRookie** [apache/skywalking#9769] [Q&A] [answered]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

Can nginx-lua-plugin set the sample rate?

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment, marked as answer]
You could use LUA to write some codes to sample, if itis sampled, call `skywalking_tracer:start("upstream service")`. NOTICE, `skywalking_tracer:finish()` should match the same sampling condition. Otherwise, the tracing context leaks.

**Reply by HackerRookie** [reply]
👌
